package.json

```diff
@@ -133,7 +133,7 @@
     "@types/react-dom": "^18.3.0",
     "@types/react-test-renderer": "^18.3.0",
     "@vitejs/plugin-vue": "^2.3.4",
-    "aws-amplify": "^6.9.0",
+    "aws-amplify": "6.14.2",
```

Comment:
Why pinning and also wouldn't we want the version to at least be the minimum specified in the peer deps of the exported packages?

Reply:
Was waiting for JS release to go out, all packages should be updated now including lock file

Reply:
EDIT: Jest isn't happy with `react-native` package due to the export path re-order. Jest needs `require` to come before `react-native`. However metro needs `react-native` to come first.

Pinning `aws-amplify` version to a one before the patch as a workaround for the time being